### ISSUE TITLE
Vertex AI: Context in Chat Completion 

### ISSUE THREAD

**Issue by vsumit89** [tmc/langchaingo#280]
There's no option for adding context in vertex AI API

**Comment by redanesconde** [comment]
should be solved in this PR #299. 

see how to implement:
```


	//systemChatMessage should be the 1st one in the list
	prompt := schema.SystemChatMessage{
		Content: "Hello, I am a bot and my name is Usher. I am here to help you with your business. What would you like to do today?",
	}
	question := schema.HumanChatMessage{
		Content: "What is your name?",
	}

	fmt.Println("---ASK---")
	fmt.Println(question.GetContent())

	messages := []schema.ChatMessage{prompt, question}
	//messages = append(messages, question)

	start := time.Now()
	completion, err := llm.Call(ctx, messages)
```

**Comment by tmc** [comment]
Yes this can be considered solved now.
